rascal-lsp/src/main/rascal/demo/lang/pico/LanguageServer.rsc

```diff
@@ -36,17 +36,22 @@ module demo::lang::pico::LanguageServer
 import util::LanguageServer;
 import util::IDEServices;
 import ParseTree;
+import util::ParseErrorRecovery;
 import util::Reflective;
 import lang::pico::\syntax::Main;
 
+private Tree picoParser(str input, loc origin) {
+    return parse(#start[Program], input, origin, allowRecovery=true, filters={createParseErrorFilter(false)});
+}
+
 @synopsis{A language server is simply a set of ((LanguageService))s.}
 @description{
 Each ((LanguageService)) for pico is implemented as a function.
 Here we group all services such that the LSP server can link them
 with the ((Language)) definition later.
 }
 set[LanguageService] picoLanguageServer() = {
```

Comment:
since this one is also used for the other tests, I would like to think of a way to still test that without error recovery on, things also keep working.